packages/mui-material-next/src/Divider/Divider.tsx

```diff
@@ -54,48 +55,52 @@ const DividerRoot = styled('div', {
         styles.textAlignLeft,
     ];
   },
-})(
-  ({ theme, ownerState }) => ({
-    margin: 0, // Reset browser default style.
-    flexShrink: 0,
-    borderWidth: 0,
-    borderStyle: 'solid',
-    borderColor: (theme.vars || theme).palette.divider,
-    borderBottomWidth: 'thin',
-    ...(ownerState.absolute && {
-      position: 'absolute',
-      bottom: 0,
-      left: 0,
-      width: '100%',
-    }),
-    ...(ownerState.light && {
-      borderColor: theme.vars
-        ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)`
-        : alpha(theme.palette.divider, 0.08),
-    }),
-    ...(ownerState.variant === 'inset' && {
-      marginLeft: 72,
-    }),
-    ...(ownerState.variant === 'middle' &&
-      ownerState.orientation === 'horizontal' && {
-        marginLeft: theme.spacing(2),
-        marginRight: theme.spacing(2),
+})<{ ownerState: DividerOwnerState }>(
+  ({ theme, ownerState }) => {
+    const { vars: tokens } = theme;
+
+    return {
+      margin: 0, // Reset browser default style.
+      flexShrink: 0,
+      borderWidth: 0,
+      borderStyle: 'solid',
+      borderColor: tokens.sys.color.outlineVariant,
+      borderBottomWidth: 'thin',
+      ...(ownerState.absolute && {
+        position: 'absolute',
+        bottom: 0,
+        left: 0,
+        width: '100%',
       }),
-    ...(ownerState.variant === 'middle' &&
-      ownerState.orientation === 'vertical' && {
-        marginTop: theme.spacing(1),
-        marginBottom: theme.spacing(1),
+      ...(ownerState.light && {
+        borderColor: theme.vars
+          ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)`
+          : alpha(theme.palette.divider, 0.08),
       }),
-    ...(ownerState.orientation === 'vertical' && {
-      height: '100%',
-      borderBottomWidth: 0,
-      borderRightWidth: 'thin',
-    }),
-    ...(ownerState.flexItem && {
-      alignSelf: 'stretch',
-      height: 'auto',
-    }),
-  }),
+      ...(ownerState.variant === 'inset' && {
+        marginLeft: 72,
+      }),
+      ...(ownerState.variant === 'middle' &&
+        ownerState.orientation === 'horizontal' && {
+          marginLeft: theme.spacing(2),
+          marginRight: theme.spacing(2),
+        }),
+      ...(ownerState.variant === 'middle' &&
+        ownerState.orientation === 'vertical' && {
+          marginTop: theme.spacing(1),
+          marginBottom: theme.spacing(1),
```

Comment:
Material You specs specify 16dp (16px) for the middle inset variants. Let's use `16px` directly instead of the theme spacing.

@mnajdova I think we still need to figure out how theme spacing (and similar concepts) will work alongside Material You tokens. Do you think we should focus on this soon? Or has this already been discussed?

Reply:
I updated the values based on the Figma file created by MD3 and hardcoded them (set 16px). If any changes are needed, please let me know so they can be applied.